### ISSUE TITLE
Could not find a part of the path

### ISSUE THREAD

**Issue by nickharrigan** [duplicati/duplicati#4092]
- [x] I have searched open and closed issues for duplicates.

----------------------------------------

## Environment info

- **Duplicati version**: 2.0.5.1_beta_2020-01-18
- **Operating system**: Windows 10
- **Backend**: Google Drive, but I have now downloaded folder and files to my local computer

## Description

I am trying to restore contents of folder 'C:\Users\<username>\'

I get the error message "Could not find a part of the path 'C:\Users\<username>\.btfs\blocks\QV\CIQOHMGEIKMPYHAUTL57JSEZN64SIJ5OIHSGJG4TJSSJLGI3PBJLQVI.data'.

## Steps to reproduce

1. Menu: Restore
2. Menu: Direct restore from backup files
3. Next >
4. Storage type... local folder or drive; choose the folderpath where the duplicati back up files are
5. Test connection ... "Connection worked!"
6. Next >
7. Passphrase ... [accepted]
8. Connect ...
9. [Takes a long time, but eventually comes to a page where you can load back up]
10. several more steps...
11. Error message "Could not find part of the path..."

- **Actual result**:
Folder where restoring files is empty. Get error message mentioned above.
- **Expected result**:
Restores my files

## Screenshots

![image](https://user-images.githubusercontent.com/17229804/74101660-5ca65580-4b90-11ea-97cb-2bc838ced6ef.png)

![image](https://user-images.githubusercontent.com/17229804/74101663-69c34480-4b90-11ea-986a-9298ada6f099.png)

## Debug log

```
Complete log  
{
  "MainOperation": "Repair",
  "RecreateDatabaseResults": {
    "MainOperation": "Repair",
    "ParsedResult": "Success",
    "Version": "2.0.5.1 (2.0.5.1_beta_2020-01-18)",
    "EndTime": "2020-02-09T11:21:49.1898011Z",
    "BeginTime": "2020-02-09T11:02:36.8228869Z",
    "Duration": "00:19:12.3669142",
    "MessagesActualLength": 0,
    "WarningsActualLength": 0,
    "ErrorsActualLength": 0,
    "Messages": null,
    "Warnings": null,
    "Errors": null,
    "BackendStatistics": {
      "RemoteCalls": 2070,
      "BytesUploaded": 0,
      "BytesDownloaded": 1232630785,
      "FilesUploaded": 0,
      "FilesDownloaded": 2069,
      "FilesDeleted": 0,
      "FoldersCreated": 0,
      "RetryAttempts": 0,
      "UnknownFileSize": 0,
      "UnknownFileCount": 0,
      "KnownFileCount": 0,
      "KnownFileSize": 0,
      "LastBackupDate": "0001-01-01T00:00:00",
      "BackupListCount": 0,
      "TotalQuotaSpace": 0,
      "FreeQuotaSpace": 0,
      "AssignedQuotaSpace": 0,
      "ReportedQuotaError": false,
      "ReportedQuotaWarning": false,
      "MainOperation": "Repair",
      "ParsedResult": "Success",
      "Version": "2.0.5.1 (2.0.5.1_beta_2020-01-18)",
      "EndTime": "0001-01-01T00:00:00",
      "BeginTime": "2020-02-09T11:02:36.7839878Z",
      "Duration": "00:00:00",
      "MessagesActualLength": 0,
      "WarningsActualLength": 0,
      "ErrorsActualLength": 0,
      "Messages": null,
      "Warnings": null,
      "Errors": null
    }
  },
  "ParsedResult": "Success",
  "Version": "2.0.5.1 (2.0.5.1_beta_2020-01-18)",
  "EndTime": "2020-02-09T11:21:49.5718059Z",
  "BeginTime": "2020-02-09T11:02:36.7839878Z",
  "Duration": "00:19:12.7878181",
  "MessagesActualLength": 4146,
  "WarningsActualLength": 0,
  "ErrorsActualLength": 0,
  "Messages": [
    "2020-02-09 22:02:36 +11 - [Information-Duplicati.Library.Main.Controller-StartingOperation]: The operation Repair has started",
    "2020-02-09 22:02:37 +11 - [Information-Duplicati.Library.Main.BasicResults-BackendEvent]: Backend event: List - Started:  ()",
    "2020-02-09 22:02:37 +11 - [Information-Duplicati.Library.Main.BasicResults-BackendEvent]: Backend event: List - Completed:  (4.00 KB)",
    "2020-02-09 22:03:21 +11 - [Information-Duplicati.Library.Main.Operation.RecreateDatabaseHandler-RebuildStarted]: Rebuild database started, downloading 8 filelists",
    "2020-02-09 22:03:21 +11 - [Information-Duplicati.Library.Main.BasicResults-BackendEvent]: Backend event: Get - Started: duplicati-20191025T033320Z.dlist.zip.aes (37.74 MB)",
    "2020-02-09 22:03:21 +11 - [Information-Duplicati.Library.Main.BasicResults-BackendEvent]: Backend event: Get - Completed: duplicati-20191025T033320Z.dlist.zip.aes (37.74 MB)",
    "2020-02-09 22:03:21 +11 - [Information-Duplicati.Library.Main.BasicResults-BackendEvent]: Backend event: Get - Started: duplicati-20191126T140000Z.dlist.zip.aes (38.25 MB)",
    "2020-02-09 22:03:22 +11 - [Information-Duplicati.Library.Main.BasicResults-BackendEvent]: Backend event: Get - Completed: duplicati-20191126T140000Z.dlist.zip.aes (38.25 MB)",
    "2020-02-09 22:03:55 +11 - [Information-Duplicati.Library.Main.BasicResults-BackendEvent]: Backend event: Get - Started: duplicati-20200101T065202Z.dlist.zip.aes (42.56 MB)",
    "2020-02-09 22:03:56 +11 - [Information-Duplicati.Library.Main.BasicResults-BackendEvent]: Backend event: Get - Completed: duplicati-20200101T065202Z.dlist.zip.aes (42.56 MB)",
    "2020-02-09 22:04:15 +11 - [Information-Duplicati.Library.Main.BasicResults-BackendEvent]: Backend event: Get - Started: duplicati-20200110T024803Z.dlist.zip.aes (42.56 MB)",
    "2020-02-09 22:04:16 +11 - [Information-Duplicati.Library.Main.BasicResults-BackendEvent]: Backend event: Get - Completed: duplicati-20200110T024803Z.dlist.zip.aes (42.56 MB)",
    "2020-02-09 22:04:36 +11 - [Information-Duplicati.Library.Main.BasicResults-BackendEvent]: Backend event: Get - Started: duplicati-20200118T052035Z.dlist.zip.aes (39.60 MB)",
    "2020-02-09 22:04:36 +11 - [Information-Duplicati.Library.Main.BasicResults-BackendEvent]: Backend event: Get - Completed: duplicati-20200118T052035Z.dlist.zip.aes (39.60 MB)",
    "2020-02-09 22:04:56 +11 - [Information-Duplicati.Library.Main.BasicResults-BackendEvent]: Backend event: Get - Started: duplicati-20200128T140001Z.dlist.zip.aes (39.67 MB)",
    "2020-02-09 22:04:56 +11 - [Information-Duplicati.Library.Main.BasicResults-BackendEvent]: Backend event: Get - Completed: duplicati-20200128T140001Z.dlist.zip.aes (39.67 MB)",
    "2020-02-09 22:05:19 +11 - [Information-Duplicati.Library.Main.BasicResults-BackendEvent]: Backend event: Get - Started: duplicati-20200130T140000Z.dlist.zip.aes (39.69 MB)",
    "2020-02-09 22:05:19 +11 - [Information-Duplicati.Library.Main.BasicResults-BackendEvent]: Backend event: Get - Completed: duplicati-20200130T140000Z.dlist.zip.aes (39.69 MB)",
    "2020-02-09 22:05:38 +11 - [Information-Duplicati.Library.Main.BasicResults-BackendEvent]: Backend event: Get - Started: duplicati-20200131T140002Z.dlist.zip.aes (39.69 MB)",
    "2020-02-09 22:05:38 +11 - [Information-Duplicati.Library.Main.BasicResults-BackendEvent]: Backend event: Get - Completed: duplicati-20200131T140002Z.dlist.zip.aes (39.69 MB)"
  ],
  "Warnings": [],
  "Errors": [],
  "BackendStatistics": {
    "RemoteCalls": 2070,
    "BytesUploaded": 0,
    "BytesDownloaded": 1232630785,
    "FilesUploaded": 0,
    "FilesDownloaded": 2069,
    "FilesDeleted": 0,
    "FoldersCreated": 0,
    "RetryAttempts": 0,
    "UnknownFileSize": 0,
    "UnknownFileCount": 0,
    "KnownFileCount": 0,
    "KnownFileSize": 0,
    "LastBackupDate": "0001-01-01T00:00:00",
    "BackupListCount": 0,
    "TotalQuotaSpace": 0,
    "FreeQuotaSpace": 0,
    "AssignedQuotaSpace": 0,
    "ReportedQuotaError": false,
    "ReportedQuotaWarning": false,
    "MainOperation": "Repair",
    "ParsedResult": "Success",
    "Version": "2.0.5.1 (2.0.5.1_beta_2020-01-18)",
    "EndTime": "0001-01-01T00:00:00",
    "BeginTime": "2020-02-09T11:02:36.7839878Z",
    "Duration": "00:00:00",
    "MessagesActualLength": 0,
    "WarningsActualLength": 0,
    "ErrorsActualLength": 0,
    "Messages": null,
    "Warnings": null,
    "Errors": null
  }
}
```

**Comment by warwickmm** [comment]
Not sure if `CIQOHMGEIKMPYHAUTL57JSEZN64SIJ5OIHSGJG4TJSSJLGI3PBJLQVI.data` was an empty file, but if it was, this is fixed in pull request #4245.

**Comment by warwickmm** [comment]
I'm guessing that this is closed by pull request #4245.  If you find this issue still occurs, feel free to reopen this.
